### ISSUE TITLE
Potplayer Integration

### ISSUE THREAD

**Issue by element209** [FredTungsten/ScriptPlayer#159]
Hello and thank you for making script player!

I hope your weekend is going well.

Please, can you make support for potplayer? I like that the player can view 3D videos on PC.

Thank you!

**Comment by FredTungsten** [comment]
You're welcome

I still have some open requests for other players, but in the meantime - do you have a link to a public API for that player? 
Can't integrate it without one.

**Comment by FredTungsten** [comment]
Closed for apparent lack of interest

**Comment by IIEleven11** [comment]
Potplayers 3d to 2d ability is second to none. I've looked into it and I dont think  it is open source, it's a chinese based program. What I did find is that it accepts angelscript to broaden its functionality. http://www.angelcode.com/angelscript/sdk/docs/manual/index.html
I am currently in college for Computer Science and am willing to help (if you want or need it) but this type of stuff (extensions, API's, integration) is new to me. I kind of need to be pointed in the right direction I think. As in what too google basically. 

I also found these command line switches buried in their documentation, I think maybe they could have something to do with a web interface? I could be totally wrong though. 
/user_agent="ua":Sets user agent header for HTTP(S) requests.
/referer="ref"  :Sets referer header for HTTP(S) requests.
/headers="hdrs" :Appends the specified header(s) to HTTP(S) requests.
